### PR TITLE
fix(ma-43): update of fullcalendar dependency to 5.3.x

### PR DESCRIPTION
**Pull request by HassanCehef**
This PR bumps fullcalendar code, vue and daygrid from 4.4.x to 5.3.x

This major version bump required several changes:
- options are specified in a different way
- several options were renamed
- remove double load of events at first load (mounted function)

Closing the three PRs from dependabot-preview, since the updates are all done here
closes #571
closes #564 
closes #563

### CODE REVIEW

**Review by WikiRik — APPROVED** [review]

File: src/views/dashboard/Calendar.vue [hunk 1]
```diff
@@ -142,16 +145,11 @@ export default {
         statutoryPromise,
         onlinePromise
       ]).then(([regular, statutory, online]) => {
-        this.events = [...regular, ...statutory, ...online]
+        this.calendarOptions.events = [...regular, ...statutory, ...online]
         this.isLoading = false
       })
     }
   },
-  mounted () {
```
Comment (serge1peshcoff) [inline src/views/dashboard/Calendar.vue:150]: @HassanCehef won't it stop the events list from being loaded?

Comment (WikiRik) [reply src/views/dashboard/Calendar.vue:150]: He locally made an event and that was being loaded, so no

Comment (HassanCehef) [reply src/views/dashboard/Calendar.vue:150]: The fetch function was called twice due to this "mounted".

Comment (serge1peshcoff) [reply src/views/dashboard/Calendar.vue:150]: okay, if it works then this makes sense indeed

**Pull request merged**